app/sell/index.js

```diff
@@ -47,6 +48,14 @@ const Sell = () => {
 
     const profileData = useSelector(selectProfileData)
 
+    const [refreshing, setRefreshing] = useState(false)
+
+    const onRefresh = useCallback(() => {
```

Comment:
Would it be possible to make the refresh event also reset the 60 second timer?

Reply:
My last comment might have been unclear, but we don't want manually refreshing to **stop** the periodic refresh, only to **reset** it. So maybe make this code a separate function:
```
        setTimeInterval(
            setInterval(() => {
                dispatch(getOrders)
            }, 60000)
        )
```
that gets called both within the useEffect hook and at the end of onRefresh?